### PR TITLE
DOCS-15841 quickstart topology update

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15841-quickstart-and-like-topologies-v1.2/quickstart/#define-a-source-and-a-destination-cluster)

[JIRA](https://jira.mongodb.org/browse/DOCS-15841)

[build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=63ee79b5dcefe97797fe7295)

### CODE REVIEW

File: source/quickstart.txt [hunk 1]
```diff
@@ -45,16 +45,10 @@ Follow the instructions below to setup {+c2c-product-name+}.
...
-        the destination cluster must also be a replica set. The number
-        of nodes in the destination replica set does not have to equal
-        the number of nodes in the source replica set.
-      - If the source cluster is :ref:`sharded  <sharding-background>`,
-        the destination cluster must have the same number of shards and
-        the same shard key. 
+      This Quickstart works when the destination cluster and the source
+      cluster are both replica sets. If you want to sync from a replica
+      set to a sharded cluster, or between sharded clusters,
+      see: :ref:`c2c-sharded-clusters`
```
Comment (jason-price-mongodb) [inline source/quickstart.txt:51]: Missing period

File: source/quickstart.txt [hunk 2]
```diff
@@ -45,16 +45,10 @@ Follow the instructions below to setup {+c2c-product-name+}.
...
-      
-      - If the source cluster is a :ref:`replica  set <replication>`,
-        the destination cluster must also be a replica set. The number
-        of nodes in the destination replica set does not have to equal
-        the number of nodes in the source replica set.
-      - If the source cluster is :ref:`sharded  <sharding-background>`,
-        the destination cluster must have the same number of shards and
-        the same shard key. 
+      This Quickstart works when the destination cluster and the source
+      cluster are both replica sets. If you want to sync from a replica
```
Comment (jason-price-mongodb) [inline source/quickstart.txt:49]: "To sync ..."

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM with comments. Cheers.

**Review by davemungo — COMMENTED** [review]
@jason-price-mongodb  thanks for the feedback, sending for external review next

**Comment by davemungo** [comment]
@alan-zheng-mdb  This PR updates the quickstart page. Would you have a look?

**Review by alan-zheng-mdb — APPROVED** [review]
lgtm

**Pull request merged**